docker-compose.yaml

```diff
@@ -33,6 +33,7 @@ services:
       # Below volume mappings are required for DISPLAY settings by heart disease GUI client
       - /tmp/.X11-unix:/tmp/.X11-unix
       - ~/.Xauthority:/root/.Xauthority
+      - ./tests/validation_suite:/project/TrustedComputeFramework/tests/validation_suite
```

Comment:
Do we need really this? We already Copy files from host to container in Docker file

Reply:
It has been removed.